src/commands/ui.rs

```diff
@@ -111,17 +113,21 @@ fn _get_local_ui_url(cmd: &UI, cfg: &edgedb_tokio::Config) -> anyhow::Result<Str
             match open_url(&url).map(|r| r.status()) {
                 Ok(reqwest::StatusCode::OK) => {}
                 Ok(reqwest::StatusCode::NOT_FOUND) => {
-                    print::error("Web UI not served correctly by specified EdgeDB server.");
+                    print::error(concatcp!(
+                        "Web UI not served correctly by specified ",
+                        BRANDING,
+                        " server."
+                    ));
                     print::echo!(
-                        "  If you have EdgeDB 2.0 and above, try running the \
+                        "  If you have {BRANDING} 2.0 and above, try running the \
```

Comment:
Let's drop the first half of this sentence. "Gel 2.0" doesn't really exist and nobody is running EdgeDB 1.0 anymore.

Reply:
```suggestion
                        "  Try running the \
```